TUM Campus App/Cafeteria/Dish.swift

```diff
@@ -54,7 +56,7 @@ struct Dish: Decodable {
      */
 
     let name: String
-    let prices: [String: Price]
+    let prices: [String: Price?]
```

Comment:
This will prevent some Dishes from being parsed. See: https://github.com/TUM-Dev/eat-api/pull/26

Reply:
Got it. The problem is that when this is not optional, my JSON decoder fails for some reason. not sure why...

Reply:
Why will this prevent dishes from being parsed? The link you posted shows that some dished have prices that look like 
"prices": {
                            "students": null,
                            "staff": null,
                            "guests": null
                        }
which means in the code above Price should be optional right?
Also when I remove the optional, my JSON decoder can no longer decode it...
So I see no solution for this besides leaving it optional, maybe you can help me here?

Reply:
My bad. I thought it was the other way around. This change should already be present in main. So a rebase / merge might a good idea before merging this.